### PR TITLE
Fixes #30098 - Drop trends

### PR DESCRIPTION
**Pull request by ezr-ondrej**
They've killed kenny! And trends! Bastartds!
Extraction PR: theforeman/foreman_statistics#9

### CODE REVIEW

**Comment by ohadlevy** [comment]
just wondering, did we lose the react router navigation and now every time you go from any page to statistics page its a full page reload?

**Comment by ezr-ondrej** [comment]
> just wondering, did we lose the react router navigation and now every time you go from any page to statistics page its a full page reload?

Hm, I'm probably not the right person to answer this, but I guess that's we are at because of the separate React controllers per plugin. Maybe @amirfefer would know?

File: app/controllers/api/v2/trends_controller.rb [hunk 1]
```diff
@@ -45,7 +50,12 @@ def resource_scope_for_index(options = {})
       end
 
       def resource_scope(options = {})
-        @resource_scope ||= scope_for(Trend.types, options)
+        @resource_scope ||= scope_for(ForemanStatistics::Trend.types, options)
+      end
+
+      def ensure_statistics_plugin
+        plugin = Foreman::Plugin.find(:foreman_statistics).nil?
+        not_found(_('For access to /trends API you need to install Foreman Statistics plugin')) unless plugin
```
Comment (shiramax) [inline app/controllers/api/v2/trends_controller.rb]: I believe it should be if instead of unless.

**Review by shiramax — CHANGES_REQUESTED** [review]

File: app/controllers/api/v2/trends_controller.rb (hunk 1 above)
Comment (ezr-ondrej) [reply app/controllers/api/v2/trends_controller.rb]: right :facepalm:

**Comment by ezr-ondrej** [comment]
This is splitted into following PRs:
#7799, #7800, #7801, #7802, #7803

**Pull request closed without merging**